i8_terminal/commands/screen/screen_losers.py

```diff
@@ -22,8 +25,18 @@
     "--view_name", "view_name", "-v", type=MetricViewParamType(), help="Metric view name in configuration file."
 )
 @click.option("--export", "export_path", "-e", help="Filename to export the output to.")
+@click.option("--metrics", "metrics", "-m", type=MetricIdentifierParamType(), help="Additional metrices to analyse.")
```

Comment:
Please change the help value to something like this:  "Comma-separated list of additional metrics."